levelkdev/swapr-dapp
# Feature/1471 bsc chain

# Summary

**Partially** Fixes #1471

- Add BSC chain suppport.
- Add PancakeSwap as a usable exchange.
- Add SushiSwap to BSC chain.

Bridge is not functional yet. Tokens are not yet mapped.

# To Test
Ignore the automated deployment.

1. Locally, clean your environment (node_modules & yarn.lock). Link the sdk from [this branch](https://github.com/levelkdev/swapr-sdk/tree/feature/1471-BSC-network)
2. Start the application and test the functionalities
3. Test the several functionalities. You should be able to switch to the BSC chain and be able to swap.

Note: The SDK dependency will be reverted to a standard format once we have an SDK release.
